R/extractDocumentation.R

```diff
@@ -83,6 +89,23 @@ extractDocumentation <- function(path, start_type=NULL, comment="*'") {
       x[1] <- sub(code,"\\2",x[1])
     }
     
+    # if (!is.null(extraPage)) {
+    #   dirPath <- "doc/markdown"
+    #   if (!dir.exists(dirPath)) {
+    #   dir.create(dirPath, recursive = TRUE)
+    # }
+    #   mdFile <- paste0(dirPath, "/", extraPage, ".md")
+    #   x <- x[-1]
+    #   write(paste(extraPage, "\n=================================\nDescription\n-----------"), mdFile)
+    #   write(x, mdFile, append = TRUE)
+    # }
+     if (!is.null(extraPage)) {
```

Comment:
This part must not happen here. `extract_block` has the purpose to extract information from a code block and to return it. The creation of the markdown files happens somewhere else!